sk0-back/src/auth/jwt-payload.interface.ts

```diff
@@ -0,0 +1,6 @@
+export interface JwtPayload {
+  email: string;
+  sub: number;
+  // I just realized we don't have roles yet lol
+  // role?: string;
```

Comment:
Yes, @rolcho mentioned that we do not have roles yet, you can have it empty or add a `USER` or similar for now.

Reply:
gotcha